addons/web_tour/static/tests/tour_recorder.test.js

```diff
@@ -446,114 +363,6 @@ test("Edit contenteditable", async () => {
     expect(tourRecorder.state.steps.map((s) => s.run)).toEqual(["editor Bismillah"]);
 });
 
-test("Run custom tour", async () => {
```

Comment:
those two tests  `"Run custom tour` and `Run a custom tour twice doesn't trigger traceback` do seem still relevant if the mock server is correctly setup, don't they ?

Reply:
Running a custom tour is the same thing that running an onboarding tour. So the tour_service.test.js already test it.